modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss

```diff
@@ -50,79 +50,130 @@ body {
 
 // Buttons
 
-.btn-link:not(.dropdown-item) {
-	@include button-cp(
-		var(--transparent),
-		var(--transparent),
-		var(--primary),
-		var(--transparent),
-		var(--transparent),
-		var(--primary-d1)
-	);
-}
+.btn:not(.btn-link) {
+	position: relative;
+	z-index: 1;
+
+	&::after {
+		background-color: black;
+		border-radius: clay-enable-rounded($btn-border-radius);
+		bottom: math-sign($btn-border-width);
+		content: '';
+		left: math-sign($btn-border-width);
+		opacity: 0;
+		position: absolute;
+		right: math-sign($btn-border-width);
+		top: math-sign($btn-border-width);
+		transition: opacity 150ms ease-in-out;
+		will-change: opacity;
+		z-index: -1;
+	}
 
-.btn-outline-light {
-	@include button-cp(
-		var(--transparent),
-		var(--white),
-		var(--white),
-		var(--white),
-		var(--white),
-		var(--dark)
-	);
-}
+	&:hover,
+	&:focus {
+		&::after {
+			opacity: 0.2;
+		}
+	}
 
-.btn-outline-primary {
-	@include button-cp(
-		var(--transparent),
-		var(--primary),
-		var(--primary),
-		var(--primary-d1),
-		var(--primary-d1),
-		var(--white)
-	);
-}
+	&:active,
+	&.active {
+		&::after {
+			opacity: 0.3;
```

Comment:
Using a different opacity value instead of overwrite the alpha value of the color gives us a smoother transition